lib/mixlib/install/cli.rb

```diff
@@ -32,7 +33,9 @@ def list_versions(product_name, channel)
       option :platform_version,
         aliases: ["-l"]
       option :architecture,
-        aliases: ["-a"]
+        default: "x86_64",
+        aliases: ["-a"],
+        enum: %w{i386 s390x sparc x86_64}
```

Comment:
Can we verify that for solaris, it is expecting `x86_64` and not `i86pc`? I don't recall where that logic stands... (and theoretically it they should both return the same non-sparc package)